### PR TITLE
fix: Add aria-label to some <Th>

### PR DESCRIPTION
**Pull request by gildub**
Fix https://github.com/trustification/trustify-ui/issues/96

### CODE REVIEW

File: client/src/app/pages/sbom-list/sbom-list.tsx [hunk 1]
```diff
@@ -159,11 +159,26 @@ export const SbomList: React.FC = () => {
...
+                    {...getThProps({ columnKey: "published" })}
+                  />
+                  <Th
+                    aria-label="packages"
+                    {...getThProps({ columnKey: "packages" })}
+                  />
+                  <Th
+                    aria-label="vulnerabilities"
+                    {...getThProps({ columnKey: "vulnerabilities" })}
+                  />
```
Comment (carlosthe19916) [inline client/src/app/pages/sbom-list/sbom-list.tsx:181]: Although nice to have, your proposed change at `client/src/app/components/TableControls/TableHeaderContentWithControls.tsx` is fully solving the original issue, don't you think it is not necessary to add these changes? Some in the file below.

File: client/src/app/pages/sbom-list/sbom-list.tsx [hunk 2]
```diff
@@ -322,8 +337,11 @@ export const SbomExpandedArea: React.FC<SbomExpandedAreaProps> = ({
         <Thead>
           <Tr>
             <TableHeaderContentWithControls {...tableControls}>
-              <Th {...getThProps({ columnKey: "name" })} />
-              <Th {...getThProps({ columnKey: "version" })} />
+              <Th aria-label="name" {...getThProps({ columnKey: "name" })} />
+              <Th
+                aria-label="version"
+                {...getThProps({ columnKey: "version" })}
+              />
```
Comment (carlosthe19916) [inline client/src/app/pages/sbom-list/sbom-list.tsx:344]: same here, there should be no need to add aria-label. 
If we start adding `aria-label` to one, then we might need to add it everywhere and although looks lazy I think we could avoid adding things and keep it simple unless required

**Review by carlosthe19916 — CHANGES_REQUESTED** [review]

File: client/src/app/components/TableControls/TableHeaderContentWithControls.tsx [hunk 3]
```diff
@@ -14,13 +14,13 @@ export const TableHeaderContentWithControls: React.FC<
     {Array(numColumnsBeforeData)
       .fill(null)
       .map((_, i) => (
-        <Th key={i} />
+        <Th aria-label="before-data-{i}" key={i} />
```
[the remark this answers is not in the document]
Comment (gildub) [reply client/src/app/components/TableControls/TableHeaderContentWithControls.tsx:17]: @carlosthe19916, come-on, my approach was original isn't and it works. But let's change it to stick with common practices.

File: client/src/app/components/TableControls/TableHeaderContentWithControls.tsx [hunk 4]
```diff
@@ -14,13 +14,13 @@ export const TableHeaderContentWithControls: React.FC<
...
       ))}
     {children}
     {Array(numColumnsAfterData)
       .fill(null)
       .map((_, i) => (
-        <Th key={i} />
+        <Th aria-label="after-data-{i}" key={i} />
       ))}
   </>
```
[the remark this answers is not in the document]
Comment (gildub) [reply client/src/app/components/TableControls/TableHeaderContentWithControls.tsx:25]: @carlosthe19916, yes I agree as not only it works too but this is a case of hidden Th content.

**Review by carlosthe19916 — APPROVED** [review]

**Pull request merged**